cache-config/testing/docker/trafficserver/run.sh

```diff
@@ -57,31 +83,42 @@ else
   source scl_source enable gcc-toolset-9
 fi
 
-initBuildArea
+cd /root
+# prep build environment
+[ -e rpmbuild ] && rm -rf rpmbuild
+[ ! -e rpmbuild ] || { echo "Failed to clean up rpm build directory 'rpmbuild': $?" >&2; exit 1; }
+mkdir -p rpmbuild/{BUILD,BUILDROOT,RPMS,SPECS,SOURCES,SRPMS} || die "Failed to create build directory '$RPMBUILD': $?"
 
 cd /root/rpmbuild/SOURCES
 # clone the trafficserver repo
-git clone https://github.com/apache/trafficserver.git
+git clone https://github.com/apache/trafficserver.git --branch $ATS_VERSION || die "Failed to fetch the ATS Source"
+cp /traffic_server_jemalloc .
+
+# patch in the astats plugin
+(cp -far /astats_over_http /root/rpmbuild/SOURCES/trafficserver/plugins/astats_over_http
```

Comment:
`-a` implies `-r`, so `cp -far` can be just `cp -fa`.

Reply:
changed.